src/index.html

```diff
@@ -6,16 +6,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>JSHeroes Bootcamp</title>
     <link rel="stylesheet" href="style.css" />
+    <script src='script.js' defer></script>
   </head>
   <body>
     <header>
       <h1>Welcome to the JSHeroes Bootcamp!</h1>
     </header>
     <main>
-      <ul class="repo-list">
-        <!-- repo list -->
-      </ul>
+      <header>
+        <h1>GitHub Explorer</h1>
+      </header>
```

Comment:
I would replace the existing header/h1 from outside of `<main>` with this, to leave room inside the main tag for the search form and the list element. We can also use the section as you suggested here, but maybe we should introduce it only when the main tag needs to be split in two sections (so when introducing the form)

Reply:
Yeah, it doesn't make sense to have it twice. :) Good catch